app/controllers/friendships_controller.rb

```diff
@@ -3,14 +3,14 @@
 class FriendshipsController < ApplicationController
   def create
     user = User.find(params[:user_id])
-    return current_user.send_friend_request(user) if user
+    current_user.send_friend_request(user) if user
```

Comment:
You must add an additional row.
Read the indication from [milestone6](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/), please